From: Ben Hutchings <ben@decadent.org.uk>
Subject: mgag200: Disable autoload
Date: Mon, 24 Dec 2012 00:10:05 +0000
Forwarded: not-needed

This kernel driver is incompatible with older versions of
xserver-xorg-video-mga, so do not enable it by default.  It will be
enabled by a modprobe configuration file installed by
xserver-xorg-video-modesetting.

--- a/drivers/gpu/drm/mgag200/mgag200_drv.c
+++ b/drivers/gpu/drm/mgag200/mgag200_drv.c
@@ -21,7 +21,7 @@
  * which then performs further device association and calls our graphics init
  * functions
  */
-int mgag200_modeset = -1;
+int mgag200_modeset = 0;
 
 MODULE_PARM_DESC(modeset, "Disable/Enable modesetting");
 module_param_named(modeset, mgag200_modeset, int, 0400);
